### PR TITLE
Refactor for version 2.0 (learning modules app)

### PR DESCRIPTION
**Pull request by sugaroverflow**
## Changes
- refactored code to remove examples and starter kit files
- rolled back to creating a List and Card for each component type
- simplified the backend to frontend data connection (removed routers and controllers for now)
- module - phase - tasks is working but tasks are incomplete
- used materialUI grid system instead because it has breakpoints
- added back simplified controllers and routers - using get() methods only 
- added back a moduleModel - even though the code is mongoose, we can update that later. 
## Component Structure 

App -> Module -> Phase -> Task -> Step || Guidance

### CODE REVIEW

**Comment by sugaroverflow** [comment]
@xiaoweiguang I made some updates - added routing and controllers - they're very simple, but we can add to them as the functionality grows. 

Please take a few minutes to review on Thursday. Thank you!

File: src/backend/server.js [hunk 1]
```diff
@@ -1,56 +1,11 @@
 const express = require('express');
-const os = require('os');
-const mongoose = require('mongoose');
-const bodyParser = require('body-parser');
 
 const app = express();
-app.use((req, res, next) => {
-  res.header('Access-Control-Allow-Origin', '*');
-  res.header(
```
Comment (xiaoweiguang) [inline src/backend/server.js]: Please add this header back.

**Review by xiaoweiguang — DISMISSED** [review]
All is good except one change.

File: src/backend/server.js (hunk 1 above)
Comment (sugaroverflow) [reply src/backend/server.js]: gotcha, thanks!

**Comment by sugaroverflow** [comment]
Updates made! I will merge once the build passes.

**Pull request merged**